packages/scene-composer/src/components/panels/__snapshots__/ComponentEditor.spec.tsx.snap

```diff
@@ -29,6 +39,16 @@ exports[`ComponentEditor renders correct component should render the "Camera" ed
 </div>
 `;
 
+exports[`ComponentEditor renders correct component should render the "DataBinding" editor correctly 1`] = `
+<div>
+  <div
+    component="[object Object]"
```

Comment:
Snapshot is not comparing correctly, for example different node or component settings won't change the shapshot, and thus you have a false positive.

Reply:
will fix